projects/dart/Dockerfile

```diff
@@ -14,8 +14,8 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-builder:ubuntu-24-04
-RUN apt-get update && apt install -y g++-multilib git python curl
+FROM gcr.io/oss-fuzz-base/base-builder
```

Comment:
Can we keep `ubuntu-24-04`?

Reply:
https://github.com/google/oss-fuzz/pull/14572/commits/d14789b0d5f21cdbb371941353358b76c9bce145